pkgs/misc/sagetex/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, pkgs, lib, ... }:
```

Individual derivations shouldn't reference `pkgs`, but instead the packages it needs.
```suggestion
{ lib
, stdenv
, fetchurl
}:
```